app/assets/javascripts/discourse/app/controllers/create-account.js

```diff
@@ -101,10 +101,12 @@ export default Controller.extend(
 
     @discourseComputed
     fullnameRequired() {
-      return (
-        this.get("siteSettings.full_name_required") ||
-        this.get("siteSettings.enable_names")
-      );
+      return this.get("siteSettings.full_name_required");
```

Comment:
Since you removed the `||` this is effectively an alias to the site setting now. I think you could remove both of these computed properties, and in your template check the `siteSettings.full_name_required` and `siteSettings.enable_names` directly.